grammars/css.cson

```diff
@@ -749,30 +1823,121 @@
     'beginCaptures':
       '0':
         'name': 'punctuation.definition.string.begin.css'
-    'end': '"'
+    'end': '"|(?<!\\\\)(?=$|\\n)'
     'endCaptures':
       '0':
         'name': 'punctuation.definition.string.end.css'
     'name': 'string.quoted.double.css'
     'patterns': [
       {
-        'match': '\\\\(\\h{1,6}|.)'
-        'name': 'constant.character.escape.css'
+        'begin': '(?:\\G|^)(?=(?:[^\\\\"]|\\\\.)+$)'
+        'end': '$'
+        'name': 'invalid.illegal.unclosed.string.css'
+        'patterns': [
+          {
+            'include': '#escapes'
+          }
+        ]
+      }
+      {
+        'include': '#escapes'
       }
     ]
   'string-single':
     'begin': '\''
     'beginCaptures':
       '0':
         'name': 'punctuation.definition.string.begin.css'
-    'end': '\''
+    'end': "'|(?<!\\\\)(?=$|\\n)"
     'endCaptures':
       '0':
         'name': 'punctuation.definition.string.end.css'
     'name': 'string.quoted.single.css'
     'patterns': [
       {
-        'match': '\\\\(\\h{1,6}|.)'
-        'name': 'constant.character.escape.css'
+        'begin': "(?:\\G|^)(?=(?:[^\\\\']|\\\\.)+$)"
+        'end': '$'
+        'name': 'invalid.illegal.unclosed.string.css'
+        'patterns': [
+          {
+            'include': '#escapes'
+          }
+        ]
+      }
+      {
+        'include': '#escapes'
+      }
+    ]
+  'tag-names':
```

Comment:
@50Wliu I'm unsure if you're asking a question, or waiting for input, or... just pointing something out. What is it you want me to do?

The property-names list is the one most subject to change, but most additions will usually be in a prefixed form. This isn't uniform, though: browsers may introduce support for unprefixed property-names at odd intervals (or not at all).

For example, Chrome's always supported `tab-size` without a prefix, while Mozilla still requires `-moz-tab-size`. As I write this, Microsoft IE/Edge have NO support for the property, presumably because they're tab-hating lunatics hell-bent on twisting the world toward using soft-tabs (which is exactly what I'd expect of their twisted, demented, hateful minds).

Half the reason I'm writing this is to confirm if action is required here: the other 50% is me taking any excuse I can to rip on soft-tab users. =)